eskip/eskip.go

```diff
@@ -185,6 +179,85 @@ type RouteInfo struct {
 	ParseError error
 }
 
+// HasPredicate checks if the route has predicate with the given name.
+func (r *Route) HasPredicate(name string) bool {
```

Comment:
this function could be more generally useful and would better reflect what it does, if it worked on any slice of predicates, and not only on those that are on the same route. E.g:

```
func PredicatesContain(p []*Predicate, name string) bool {}
```